src/db/entity/ExecutionEntity.ts

```diff
@@ -28,6 +28,7 @@ class ExecutionEntity {
    */
   @OneToMany((_type) => RuleEntity, (rule) => rule.execution, {
     cascade: true,
+    eager: true,
```

Comment:
This is to fix the issue where nested objects aren't automatically retrieved by TypeOrm. See [this link](https://github.com/JID-2111/JID-2111/pull/172) for more details